source/includes/_rendering.md

```diff
@@ -4,27 +4,33 @@ Rendering is where modules can draw most anything on to the game screen. All 2D
 calling methods in the `Renderer` object. 3D rendering involves calling methods in the
 [`Tessellator`](#tessellator) object.
 
+<aside class="success">
+  If you want to create much more customizable rendering options, the
+  <a href="https://chattriggers.com/modules/v/Elementa">Elementa module</a>
+  is highly encouraged.
+</aside>
+
 <aside class="notice">All 2D rendering coordinates start at the top left of the screen. X increases from left to right, and Y increases from top to bottom.</aside>
 
 ## Setting up
 
 > Function to be ran everytime the game overlay is rendered
 
-```javascript
+```js
 register("renderOverlay", myRenderOverlay);
 
 function myRenderOverlay() {
 
 }
 ```
 
-Rendering has to be done every frame of the game, otherwise it will only be on the screen for one frame. There are many different render triggers, and they all start with `Render`. The most common render trigger is RenderOverlay: this trigger fires every frame with no conditions.
+Rendering has to be done every frame of the game, otherwise it will only be on the screen for one frame and . There are many different render triggers, and they all start with `Render`. The most common render trigger is RenderOverlay: this trigger fires every frame with no conditions.
```

Comment:
Forgot to remove this before committing I assume? Or just didn't finish typing.

Reply:
Oops yeah no idea what I meant to say there😅